class.jetpack-gutenberg.php

```diff
@@ -283,6 +283,22 @@ public static function get_blocks_directory() {
 		return apply_filters( 'jetpack_blocks_directory', '_inc/blocks/' );
 	}
 
+	/**
+	 * Return the Gutenberg extensions (blocks and plugins) source directory
+	 *
+	 * @return string The Gutenberg extensions source directory
+	 */
+	public static function get_blocks_source_directory() {
```

Comment:
I'm not sure we can rely on source being present:

https://github.com/Automattic/jetpack/blob/e2f23699e302f9be715f16aa107dba162fe0c73c/.svnignore#L45-L56

We spoke about keeping the copy webpack plugin merged with a webpack config in #11802, I think that will be a better approach and avoid introducing another public method.

Reply:
If we need the .json files in production, we can remove that line so .php and .json would remain in `extensions`.